pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -395,6 +418,43 @@ func (k *Bootstrapper) unpause(cfg config.ClusterConfig) error {
 	return nil
 }
 
+// ensureControlPlaneAlias adds control-plane.minikube.internal -> IP mapping in /etc/hosts
```

Comment:
If we move dual-stack support for the control plane to another PR, we can remove this  :)